src/Rule/BlankLineAfterDirective.php

```diff
@@ -36,8 +36,7 @@ public function check(Lines $lines, int $number): ?string
     {
         $lines->seek($number);
         $line = $lines->current();
-
-        if (!RstParser::isDirective($line)) {
+        if (null === $line || !RstParser::isDirective($line)) {
```

Comment:
should we add `assert(null !== $line)` instead?

Reply:
Yes, thank you 👍